runtime/libpgmath/CMakeLists.txt

```diff
@@ -50,6 +50,8 @@ set(LIBPGMATH_SYSTEM_PROCESSOR "${CMAKE_SYSTEM_PROCESSOR}")
 if ("${LIBPGMATH_SYSTEM_PROCESSOR}" STREQUAL "AMD64" OR
     "${LIBPGMATH_SYSTEM_PROCESSOR}" STREQUAL "amd64" )
     set(LIBPGMATH_SYSTEM_PROCESSOR "x86_64")
+elseif ("${LIBPGMATH_SYSTEM_PROCESSOR}" STREQUAL "ARM64")
```

Comment:
Why was this not required before?

Reply:
This is required because CMAKE sets the system processor to ARM64 on windows and aarch64 on linux. It's the same processor, so it makes sense to harmonize them here.